scripts/abins/instruments/broadening.py

```diff
@@ -528,13 +528,13 @@ def interpolated_broadening(sigma=None, points=None, bins=None,
     log_bases = {'2': 2, 'sqrt2': np.sqrt(2)}
     log_base = log_bases[spacing]
 
-    # Sample on appropriate log scale: log_b(x) = log(x) / lob(b)
-    n_kernels = int(np.ceil(np.log(max(sigma) / min(sigma)) / np.log(log_base)))
+    # Sample on appropriate log scale: log_b(x) = log(x) / log(b)
+    n_kernels = int(np.ceil(np.log(max(sigma) / min(sigma)) / np.log(log_base))) + 1
```

Comment:
whats the plus 1 for?

Reply:
Ensures there is always a higher kernel available so interpolation has two end groups to work between. The existing setup breaks if you try to interpolate to a width that is exactly equal to the highest width; in practice this happens for constant-width broadening (where only one kernel is really needed) and this PR introduces the "ideal simulation" 2-D instrument which has such a fixed width.

In principle it would be more efficient to
- fall back on non-interpolated broadening for fixed-width cases
- otherwise, only add the extra point when the rare but theoretically-possible max(sigma)==min(sigma)*factor^n_kernels case arises.

But the +1 is a simple fix and not a big overhead.